bin/template/.ipynb_checkpoints/0_main_template-checkpoint.ipynb

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
<p>Is <code>Boundary and Initial Conditions</code> the only planned subsection under <code>Model Setup</code>, or will more be added later?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>